.github/workflows/react-native-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: React Native Tests
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+    paths:
+      - 'binding/react-native/test-app/**'
+      - 'lib/common/**'
+      - 'lib/android/**'
+      - 'lib/ios/**'
+      - 'resources/keyword_files/android/**'
+      - 'resources/keyword_files/ios/**'
+      - '.github/workflows/react-native-tests.yml'
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'binding/react-native/test-app/**'
+      - 'lib/common/**'
+      - 'lib/android/**'
+      - 'lib/ios/**'
+      - 'resources/keyword_files/android/**'
+      - 'resources/keyword_files/ios/**'
+      - '.github/workflows/react-native-tests.yml'
```

Comment:
don't we want these to run whenever there's a change to `binding/react-native/**` ?

Reply:
Ah, right!